planner-api/src/main/java/pl/edu/agh/kuce/planner/balance/persistence/Balance.java

```diff
@@ -0,0 +1,54 @@
+package pl.edu.agh.kuce.planner.balance.persistence;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name="balances")
+public class Balance {
+    @Id
+    @Column(nullable = false)
+    private Integer userId;
```

Comment:
Currently we are doing it in a different way like in https://github.com/marszub/savings-planner/tree/master/planner-api/src/main/java/pl/edu/agh/kuce/planner/event/persistence

Reply:
I see, now we are using full User class instead of just user_id, I will swap this for something similar to this

@Id
@GeneratedValue
private Integer id;
    
@OneToOne
@JoinColumn(name = "user_id")
private User user;

@Column(nullable = false)
private Integer balance;